### PR TITLE
Display number of available reward points on staff page

### LINKED ISSUE #1927: Display number of available reward points
On the staff page for reward point redemption events (`rewards/reward_point_redemption_events`), the total number of currently available (granted but not yet redeemed) number of reward points should be shown.
The information should be placed in the upper right corner (in the same line as the "Create new event" button, vertically centered) with the following text: `Available reward points: <number>`.

### PR DESCRIPTION
**Pull request by olewoy**
closes #1927

### CODE REVIEW

**Review by He3lixxx — APPROVED** [review]

File: evap/rewards/templates/rewards_reward_point_redemption_events.html [hunk 1]
```diff
@@ -7,10 +7,14 @@
 
 {% block content %}
     {{ block.super }}
-
-    <p>
-        <a href="{% url 'rewards:reward_point_redemption_event_create' %}" class="btn btn-dark btn-sm">{% trans 'Create new event' %}</a>
-    </p>
+    <div class="row mb-2 align-items-center">
```
Comment (niklasmohrin) [inline evap/rewards/templates/rewards_reward_point_redemption_events.html:10]: This seems to be unused:
```suggestion
    <div class="row mb-2">
```

File: evap/rewards/templates/rewards_reward_point_redemption_events.html [hunk 2]
```diff
@@ -7,10 +7,14 @@
...
+        <div class="col">
+            <a href="{% url 'rewards:reward_point_redemption_event_create' %}" class="btn btn-dark btn-sm">{% trans 'Create new event' %}</a>
+        </div>
+        <div class="col" style="text-align: right">
```
Comment (niklasmohrin) [inline evap/rewards/templates/rewards_reward_point_redemption_events.html:14]: There is a bootstrap class for this style already:
```suggestion
        <div class="col text-end">
```

File: evap/rewards/templates/rewards_reward_point_redemption_events.html [hunk 3]
```diff
@@ -7,10 +7,14 @@
...
+            {% trans 'Available reward points' %}: {{total_points_available}}
```
Comment (niklasmohrin) [inline evap/rewards/templates/rewards_reward_point_redemption_events.html:15]: We usually indent it like this (not that you could know, we didn't really write it down anywhere):
```suggestion
            {% trans 'Available reward points' %}: {{ total_points_available }}
```

**Review by niklasmohrin — COMMENTED** [review]
Code looks good, some comments

**Review by janno42 — APPROVED** [review]
✔️ Meets requirements
✔️ UI functionality checked

**Comment by janno42** [comment]
Thanks! :shipit:

**Pull request merged**